kojak/cli.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import os
+
+
+def is_valid_path(path):
+    if not os.path.isdir(path) and not os.path.isfile(path):
```

Comment:
I don't understand this condition? How that can be not and file and not a directory?

Reply:
if a the given path is not a dir or not a file in the FS then something went wrong and the path doesn't exist. 

It's ok?

Reply:
So you can use exists function.